helix-term/src/commands/typed.rs

```diff
@@ -2337,6 +2395,13 @@ pub const TYPABLE_COMMAND_LIST: &[TypableCommand] = &[
             fun: run_shell_command,
             completer: Some(completers::directory),
         },
+       TypableCommand {
+            name: "reset-change",
+            aliases: &["reset"],
+            doc: "Undo the change at cursor position",
+            fun: reset_change,
+            completer: None,
+        },
```

Comment:
I'm not sure where to put this but I think either the name or description (or both) should say "VCS". Otherwise this might be confused with https://github.com/helix-editor/helix/issues/3364

Reply:
The reason I shy away from using `vcs` in diff related functionality is that these commands would also be used for the diffmode where we might not actually be comparing against a VCS file but rather just a different buffer/file on disk. 

I would have used something like `undo-selection` for #3364. So any command that goes back in undo history would contain `undo` and `reset` would be for the diff. I can see that the names could be potentially confusing tough. This is sort of a bikeshed issue.


For now I just changed the description so that it uses `reset` instead of `undo` (to keep terminology straight) and explicitly refer to a`diff change`. Do you think that is enough or would you rather see the name changed.
Some options I came up wiith:

* `:reset-change`: What I choose initially for this PR.
* `:reset-diff-change`: Probably most concise but a bit wordy
* `:reset-diff`: Sounds like the entire diff is reset
* `:diffget` (alias `:diffg`): Name stolen from nvim. The nice thing is that it has a cousing `:diffput` to reset the other side of the diff if we are comparing to editable files (not currently implemented)

Reply:
Thanks for your input. I renamed to command to `:reset-diff-change`.
I decided to add `:diffget` and `:diffg` as aliases. These names are quite well know from other editors, and I don't think they are not easily confused with something else.
I personally don't bind this to a keybinding so I also appreciate being able to type `:diffg` as a convenience (similar to how we have `:w` as an alias for `:write`).

I also think that it will make it clearer that this command is closely linked to a future command that changes the other side of the disk (which would alias to `:diffput`)